core/arch/instr.h

```diff
@@ -325,6 +325,26 @@ typedef enum _dr_encoding_hint_type_t {
 #endif
 } dr_encoding_hint_type_t;
 
+/** AVX-512 tuple type attributes as specified in Intel's tables. */
```

Comment:
These are internal, so don't use a Doxygen comment.

Reply:
The other enums have a doxygen comment. Sure?

Reply:
Offline.